plugins/inputs/system/PROCESSES_README.md

```diff
@@ -39,6 +40,7 @@ Linux  FreeBSD  Darwin  meaning
   R       R       R     running
   S       S       S     sleeping
   Z       Z       Z     zombie
+  X       X       X     dead
```

Comment:
Is this correct?  I think it should be `X none none` based on manpages I've found online.